compiler/optimizations/remoteValueForwarding.cpp

```diff
@@ -511,6 +544,27 @@ static void insertSerialization(FnSymbol*  fn,
   }
 }
 
+static CallExpr* findDestroyCallForArg(ArgSymbol* arg) {
+  // Scroll backwards looking for an auto-destroy call for arg
+  // in the function defining arg.
+
+  FnSymbol* fn = arg->getFunction();
+  for (Expr* stmt = fn->body->body.tail;
```

Comment:
Would it be better to use the list of SymExprs stored in ``arg``? I don't really care about the performance here. It just seems like a more straightforward way than scanning the entire function body.

Reply:
I thought it important not to find autoDestroy calls inside conditionals etc, but I debated about this. Certainly the AST pattern we have seen for this always has it at the end of the function.

Reply:
Fair point, works for me!